kolibri/plugins/user_profile/assets/src/views/ChangeFacility/ConfirmChangeFacility.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
+    <p>{{ firstLine }}</p>
+    <p>{{ secondLine }}</p>
+    <p>{{ thirdLine }}</p>
+    <BottomAppBar>
+      <slot name="buttons">
+        <KButtonGroup>
+          <KButton
+            :primary="false"
+            :text="$tr('mergeAccounts')"
+            @click="to_merge"
+          />
+          <KButton
+            :primary="true"
+            :text="coreString('continueAction')"
+            @click="to_continue"
+          />
+        </KButtonGroup>
+      </slot>
+    </BottomAppBar>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+
+  export default {
+    name: 'ConfirmChangeFacility',
+    metaInfo() {
+      return {
+        title: this.$tr('documentTitle'),
+      };
+    },
+    components: { BottomAppBar },
+
+    mixins: [commonCoreStrings],
+
+    inject: ['changeFacilityService', 'state'],
+    computed: {
+      targetFacility() {
+        return this.state.value.targetFacility;
+      },
+      role() {
+        return this.state.value.role;
+      },
+      firstLine() {
+        return this.$tr('changeFacilityInfoLine1', {
+          target_facility: this.targetFacility.name,
+        });
+      },
+      secondLine() {
+        if (this.role === 'learner') return '';
+        return this.$tr('changeFacilityInfoLine2', {
+          role: this.role,
+        });
+      },
+      thirdLine() {
+        return this.$tr('changeFacilityInfoLine3', {
+          target_facility: this.targetFacility.name,
+        });
+      },
+    },
+
+    methods: {
+      to_continue() {
+        this.changeFacilityService.send({
+          type: 'CONTINUE',
+        });
+      },
+      to_merge() {
+        this.changeFacilityService.send({
+          type: 'MERGE',
+        });
+      },
+    },
+
+    $trs: {
+      documentTitle: {
+        message: 'Change Facility',
+        context: 'Title of this step for the change facility page.',
+      },
+      mergeAccounts: {
+        message: 'Merge Accounts',
+        context: 'Button for the merge accounts between facilities button.',
+      },
+      changeFacilityInfoLine1: {
+        message:
+          'You are about to move your account and progress data to ‘{target_facility}’ learning facility. Your data will still be available to you and will also be accessible to any administrators of this learning facility.',
+        context: 'First line of text explaining what changing to another learning facility means.',
```

Comment:
I was asking rather if translators know about the context in terms of the UI feature or part of the feature like the modal. Because if not, they might see for example:

`First line of text explaining what changing to another learning facility means.`
`First line of text confirming the username and facility where the user is changing.`
`First line of text ...`
`Second line of text...`

and then I am not sure if it's possible to make sense out of it.

But would need to hear from @radinamatic I think because I'm not sure if translators have some more information available from namespaces, for example. No need to worry about it in this PR way too much, it's rather a general question to learn a bit more about how to write context strings.